Makefile

```diff
@@ -59,6 +59,13 @@ lint:
 		--enable=varcheck \
 		$(NULL)
 
+vendor: Gopkg.lock
+	dep ensure -vendor-only -v
+
+ifndef model
+	model:=/files/go/src/gitlab.cee.redhat.com/service/ocm-api-model/model
+endif
```

Comment:
There is no need to make this conditional: you can always override this make variable in the command line.